spec.html

```diff
@@ -12,32 +12,30 @@
 <script src="ecmarkup.js"></script>
 <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/8.4/styles/github.min.css">
 
-<emu-table id="table-1" caption="Well-known Intrinsic Objects">
-	<table>
-		<tbody>
-			<tr>
-				<th>Intrinsic Name</th>
-				<th>Global Name</th>
-				<th>ECMAScript Language Association</th>
-			</tr>
-			<tr>
-				<td>%Realm%</td>
-				<td><code>Realm</code></td>
-				<td>…</td>
-			</tr>
-			<tr>
-				<td>%RealmPrototype%</td>
-				<td><code>Realm.prototype</code></td>
-				<td>…</td>
-			</tr>
-		</tbody>
-	</table>
-</emu-table>
+<emu-clause id="sec-well-known-intrinsic-objects">
+  	<h1>Well-known intrinsic objects</h1>
+	<emu-table id="table-7" caption="Well-known Intrinsic Objects">
```

Comment:
This is Table 8 in ECMA-262 today. It might be easier leaving this as Table 1 or a non number identifier.

Reply:
This is so that **ecmarkup** auto‑links `<emu-xref href="#table-7" />` in this specification as local links, instead of links to <https://tc39.es/ecma262/#table-7>.

Reply:
I see. The id remains with the wrong number.

@tc39/ecma262-editors Is this something we can change in ECMA-262? I mean, having a nominal id rather than numeric?

Reply:
We could change the legacy ID (and preserve it via an oldid) but it doesn't seem particularly important to change to me.

Reply:
Well, let's keep the off-by-one then. :)